src/factories/beacon/InverterBeacon.sol

```diff
@@ -35,13 +35,6 @@ contract InverterBeacon is IInverterBeacon, ERC165, Ownable2Step {
     //--------------------------------------------------------------------------
     // Modifiers
 
-    modifier validNewMinorVersion(uint newMinorVersion) {
```

Comment:
Why did this modifier get removed? Will this check be done somewhere else?

Reply:
https://github.com/InverterNetwork/inverter-contracts/issues/432

Nuggan mentioned it in this issue, so I thought I would tackle it in one go

Reply:
Also it was needed when I used the upgrade function in the beacon constructor, although there might have been ways around that, now that I think about it

Reply:
Oh, I noticed this in my scripts PR as well and fixed it there as well 😂 (allowing 0 version, but only if it's the first one that is set in the beacon). But that's on me, I will go through merging hell and see what happens there once we get there.